config-coral.ini

```diff
@@ -13,16 +18,25 @@ Host = 0.0.0.0
 QueuePort = 8010
 QueueAuthKey = shibalba
 
+[Source_0]
+VideoPath = /repo/data/softbio_vid.mp4
+Tags = area1
+Name = Garden-Camera
+Id = cam1
```

Comment:
I tried to run amd64-usbtpu docker and frontend. On the browser, there are requests such as this path: "filename: /static/gstreamer/default/playlist.m3u8" 
but I found that there are another similar path with "cam1" instead of "default". I changed this Id to "default" and I could see the videos. I think a modification is required in the paths, whether in the frontend requests or here in the configs.

Reply:
Yes, you are right. We should change one of them or make it clear in the Readme. The path to the video includes the camera ID (to allow multiple cameras). But the frontend is still pointing at a hardcoded 'default' camera (which is set in web_gui.py). Ideally the frontend would need to get the config from the processor and use the IDs defined there. but for now I can change the camera IDs to 'default' in these config files